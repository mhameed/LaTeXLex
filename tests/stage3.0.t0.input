This is a line
That spanns over
multiple lines. \bigskip This is a line
That spanns over
multiple lines.

\bigskip \bigskip \bigskip \bigskip

This is a totally new paragraph.
That has some 
$f(x)$
at the end of it.
