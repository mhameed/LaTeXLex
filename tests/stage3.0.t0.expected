This is a line That spanns over multiple lines. 


This is a line That spanns over multiple lines.


This is a totally new paragraph. That has some  $f(x)$
at the end of it. 
